cha/implementaion/playback/decoding.tex

```diff
@@ -0,0 +1,48 @@
+\section{Decoding MP3}
+In \cnameref{cha:initial_development} we presented our initial foundation for development, namely the Google sample implementation of a universal media player app.
+In that implementation an Android implementation, \code{android.media.MediaPlayer}, is used to control the playback of audio streams.
+
+The decoding from MP3 to \ac{PCM} is already handled by the base implementation from Google samples, however we have no guarantees when actions occur, as it is left up to this already implemented system and done in correlation with playing the \ac{PCM} data.
+We replace the \code{android.media.MediaPlayer} class with our own implementation in order to obtain more control over the playback and decoding.
+
+We want more control to be able to guarantee certain characteristics about the data, such that we have the required control to properly stream the data from one device to another and perform playback commands in sync.
+These guarantees are: \mnote[inline]{what are these guarantees and why do we need them? I assume i missed some}.
```

Comment:
These hardly seem like "guarantees" and more like features. It feels like this section isn't really leading up to anything.